src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -202,7 +202,8 @@ internal sealed override DeclarationScope EffectiveScope
         {
             get
             {
-                var scope = DeclaredScope;
+                var scope = (RefKind == RefKind.Out && DeclaredScope == DeclarationScope.Unscoped) ? DeclarationScope.RefScoped : DeclaredScope;
```

Comment:
Does it change behavior to move this logic from ParameterHelpers, etc. into this accessor? Or is it just a refactoring?

Reply:
The behavior change is described in `ParameterSymbol`. Basically, `DeclaredScope` is now just the declared scope, without the implicit scope for various cases. `EffectiveScope` is still the combined scope.